### PR TITLE
[plugin.video.srgssr_ch_replay] 3.0.0

### PR DESCRIPTION
**Pull request by ManBehindMooN**
### Description
New Addon: Unofficial SRG SSR Replay (detailed explanation can be found in our repo README) 

Uses the official API from the SRG SSR group. The TV-Show-Library for all their official channels can be browsed and streamed. Over the new API more features can be implemented (Radio Podcasts and the like) in the future. This add-on has been presented to the SRG SSR group and we have been given an GO-Live by them.

Repo: https://github.com/ManBehindMooN/kodi_plugin_video_srgssr_ch_replay
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by ManBehindMooN** [comment]
@enen92 what is the time frame I could expect this branch to be merged? Thank you!

File: plugin.video.srgssr_ch_replay/addon.py [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+def _addnextpage(name, url, mode, desc, showbackground, pageNumber, channel, numberOfEpisodes, nextParam):
+    """
+    helper method to create a folder with subitems
+    """
+    directoryurl = sys.argv[0] + "?url=" + urllib.parse.quote_plus(url) + "&mode=" + str(mode) + "&showbackground=" + urllib.parse.quote_plus(showbackground) + \
+        "&page=" + str(pageNumber or "") + "&channel=" + str(channel) + "&numberOfEpisodes=" + str(numberOfEpisodes or "") + "&next=" + str(nextParam)
+    liz = xbmcgui.ListItem(name)
+    liz.setLabel2(desc)
+    #liz.setArt({'poster' : '' , 'banner' : '', 'fanart' : showbackground, 'thumb' : ''})
+    liz.setInfo(type="Video", infoLabels={"title": name, "plot": desc, "plotoutline": desc})
```
Comment (enen92) [inline plugin.video.srgssr_ch_replay/addon.py:342]: Setinfo is deprecated in nexus jfyi

**Review by enen92 — APPROVED** [review]
Looks good

File: plugin.video.srgssr_ch_replay/addon.py (hunk 1 above)
Comment (ManBehindMooN) [reply plugin.video.srgssr_ch_replay/addon.py:342]: @enen92 thank you, we will have a look at it and change it with the next pull request

Comment (enen92) [reply plugin.video.srgssr_ch_replay/addon.py:342]: Note two different code paths are required though, as the new one (using infotag member functions) is not compatible with matrix. SetInfo will probably be removed by v21

**Pull request merged**